src/test/java/io/suhan/lotto/model/executor/LottoDrawExecutorTest.java

```diff
@@ -25,7 +26,10 @@ public class LottoDrawExecutorTest {
         executor.execute();
 
         List<DrawResult> results = executor.getResults();
-        assertThat(results).hasSize(1);
-        assertThat(results.get(0).getMatchedCount()).isEqualTo(Lotto.LOTTO_SIZE);
+
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(results).hasSize(1);
+        softAssertions.assertThat(results.get(0).getMatchedCount()).isEqualTo(Lotto.LOTTO_SIZE);
+        softAssertions.assertAll();
```

Comment:
이거는 이런 방법으로 더 깔끔하게 해볼만한 것 같아요

- https://assertj.github.io/doc/#assertj-core-assertSoftly-assertions

```suggestion
        SoftAssertions.assertSoftly(softly -> {
            softly.assertThat(results).hasSize(1);
            softly.assertThat(results.get(0).getMatchedCount()).isEqualTo(Lotto.LOTTO_SIZE);
        });
```

Reply:
문서를 읽어보니 기존과 달리 마지막에 `assertAll()`을 실행할 필요가 없어서 이를 혹시나 빼먹는 상황을 사전에 방지할 수 있겠네요!
코드 스타일도 하나의 scope안에 여러 케이스들이 소속되어 있는 느낌이라 좀 더 직관적인 것 같습니다..!

반영 커밋: https://github.com/next-step/java-lotto-clean-playground/pull/141/commits/3bc67b06c9c9b03b874921cb4e629fd5f8d5f3b4